resources/views/admin/databases/index.blade.php

```diff
@@ -36,7 +36,7 @@
         <li><a href="{{ route('admin.databases.new') }}"><i class="fa fa-plus"></i></a></li>
     </ul>
     <div class="tab-content">
-        <div class="tab-pane active" id="tab_databases">
+        <div class="tab-pane {{ Request::input('tab') == 'tab_dbservers' ? '' : 'active' }}" id="tab_databases">
```

Comment:
Actually, I see the issue right there. it should be `?tab=tab_databases` not `?tab=tab_dbservers`, or the id needs to be changed.